code/script.js

```diff
@@ -303,41 +324,91 @@ const filterCharacters = (keep) => {
         `No, the person doesn't wear ${attribute}! Remove all that wears ${attribute}`
       )
     }
-  } else if (group === 'other') {
-    // Similar to the one above
-  } else {
-    if (keep) {
-      // alert popup that says something like: "Yes, the person has yellow hair! Keep all persons with yellow hair"
+    } else if (category === 'other') {
+      if (keep) {
+        alert(
+          `Yes, the person has ${attribute}! Keep all that has ${attribute}`
+        )
+    } else {
+        alert(
+        `No, the person doesn't have ${attribute}! Remove all that have ${attribute}`
+        )
+    } 
+    } else if (category === 'hair color') {
+      if (keep) {
+        alert(
+          `Yes, the person has ${value} hair color! Keep all that have ${value} hair color`
+        )
+    } else {
+        alert(
+          `No, the person doesn't have ${value} hair color! Remove all that have ${value} hair color`
+        )
+    } 
+    } else {
+      if (keep) {
+        alert(
+          `Yes, the person has ${value} eyes! Keep all that has ${value} eyes`
+       )
     } else {
-      // alert popup that says something like: "NO, the person doesnt have yellow hair! Remove all persons with yellow hair"
+        alert(
+          `No, the person doesn't have ${value} eyes! Remove all that have ${value} eyes`
+      )
     }
   }
 
```

Comment:
indentation for the above if statements, make sure the curly bracket belonging to each section is in the same _vertical_ line as statement, like this: 
```
if () {
} else if (){
  } else {
    }
```